scripts/run-integration-tests.sh

```diff
@@ -56,7 +56,7 @@ start-daemons() {
 	# Wait up to 10 seconds for Alice's swapd instance to start and deploy the swap contract
 	#
 	CONTRACT_ADDR_FILE="${SWAP_TEST_DATA_DIR}/alice/contract-address.json"
-	for _ in {1..10}; do
+	for _ in {1..60}; do
```

Comment:
Hopefully, it's not taking a full minute. You can remove the phrase "up to 10 seconds" from the comment.

Reply:
updated, I don't think it is but I'm not sure exactly how long it takes to start up so I just put this to be safe